app/src/main/res/values/strings.xml

```diff
@@ -2,6 +2,8 @@
     <string name="app_name">Morse</string>
     <string name="facebook_app_id">292575252317805</string>
     <string name="phone_number">Phone number</string>
+    <string name="com.twitter.sdk.android.CONSUMER_KEY">dmSE6yKjQbPLiB5Kyx9BF484N</string>
+    <string name="com.twitter.sdk.android.CONSUMER_SECRET">RKKodQ7VL2pEiJnVtwXGcfxbk3V15ykyLdfQcZFJ0TdZKmxOrU</string>
```

Comment:
Credentials stored in **plaintext** and upped to the repository. That very bad programming practice that should be avoided at all costs. Instead, I suggest adding something that will mask the credentials (something like an environment variable or whatever is considered best practice for Android when it comes to API Keys Management).

On the other hand, I've tested the app and noticed that the Twitter logo is too close to the upper edge of the layout. So I will make a commit to change that and try to lower it a bit down.